pkg/resource/aws/aws_instance.go

```diff
@@ -102,3 +102,10 @@ func (r *AwsInstance) TerraformType() string {
 func (r *AwsInstance) CtyValue() *cty.Value {
 	return r.CtyVal
 }
+
+func (r *AwsInstance) String() string {
```

Comment:
This should be put in `aws_instance_ext.go` file as this file is generated and should not be altered. This is gonna be deleted btw but it will be more straightforward for us if it follow the same architecture than other resources.